spec/functions/compliance_markup/hiera_backend_spec.rb

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby -S rspec
+# vim: set expandtab ts=2 sw=2:
+require 'spec_helper'
+require 'semantic_puppet'
+puppetver = SemanticPuppet::Version.parse(Puppet.version)
+requiredver = SemanticPuppet::Version.parse("20.9.0")
```

Comment:
This is no-oped because of issues with creating a working Puppet LookupContext in rspec-puppet. Need upstream helpers added.